src/scaler/daemonscaler/mod.rs

```diff
@@ -0,0 +1,1038 @@
+                                    // Start actors to reach desired replicas
+                                    Ordering::Less => {
+                                        // Right now just start on the first available host. We can be smarter about it later
+                                        Some(Command::StartActor(StartActor {
+                                            reference: self.config.actor_reference.to_owned(),
+                                            host_id: host_id.to_string(),
+                                            count: self.config.spread_config.replicas
+                                                - current_count,
+                                            model_name: self.config.model_name.to_owned(),
+                                            annotations: spreadscaler_annotations(
+                                                &spread.name,
+                                                self.id(),
+                                            ),
+                                        }))
+                                    }
+                                    // Stop actors to reach desired replicas
+                                    Ordering::Greater => Some(Command::StopActor(StopActor {
+                                        actor_id: actor_id.unwrap_or_default().to_owned(),
+                                        host_id: host_id.to_string(),
+                                        count: current_count - self.config.spread_config.replicas,
+                                        model_name: self.config.model_name.to_owned(),
+                                        annotations: spreadscaler_annotations(
+                                            &spread.name,
+                                            self.id(),
+                                        ),
+                                    })),
+                                }
+                            })
+                            .collect::<Vec<Command>>(),
+                    )
+                } else {
+                    // No hosts were eligible, so we can't attempt to add or remove actors
+                    trace!(?spread.name, "Found no eligible hosts for daemon scaler");
+                    spread_status.push(StatusInfo::failed(&format!(
+                        "Could not satisfy daemonscaler {} for {}, 0 eligible hosts found.",
+                        spread.name, self.config.actor_reference
+                    )));
+                    None
+                }
+            })
+            .flatten()
+            .collect::<Vec<Command>>();
+        trace!(?commands, "Calculated commands for actor daemon scaler");
+
+        let status = match (spread_status.is_empty(), commands.is_empty()) {
+            (true, true) => StatusInfo::ready(""),
+            (_, false) => StatusInfo::compensating(""),
+            (false, true) => StatusInfo::failed(
```

Comment:
Nit: I don't think I realized this before but this is a really odd way to identify an error case. I don't think it needs to be solved now, but we might want to reevaluate how to indicate errors